ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/MicrosoftSentinelForm.tsx

```diff
@@ -0,0 +1,529 @@
+                            id="notifier.microsoftSentinel.applicationClientId"
+                            value={values.notifier.microsoftSentinel.applicationClientId}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    <Card isFlat>
+                        <CardTitle>Authentication Method</CardTitle>
+                        <CardBody>
+                            <ToggleGroup aria-label="Default with single selectable">
+                                <ToggleGroupItem
+                                    text="Use Secret"
+                                    buttonId="use-secret"
+                                    isSelected={selectedAuthMethod === 'use-secret'}
+                                    onChange={onUpdateAuthMethod}
+                                    isDisabled={!isEditable}
+                                />
+                                <ToggleGroupItem
+                                    text="Use Client Certificate"
+                                    buttonId="use-client-cert"
+                                    isSelected={selectedAuthMethod === 'use-client-cert'}
+                                    onChange={onUpdateAuthMethod}
+                                    isDisabled={!isEditable}
+                                />
+                            </ToggleGroup>
+                            {!isCreating && isEditable && (
+                                <FormLabelGroup
+                                    label=""
+                                    fieldId="updatePassword"
+                                    helperText="Enable this option to replace currently stored credentials (if any)"
+                                    errors={errors}
+                                >
+                                    <Checkbox
+                                        label="Update authentication"
+                                        id="updatePassword"
+                                        isChecked={values.updatePassword}
+                                        onChange={(event, value) =>
+                                            onUpdateCredentialsChange(value, event)
+                                        }
+                                        onBlur={handleBlur}
+                                        isDisabled={!isEditable}
+                                    />
+                                </FormLabelGroup>
+                            )}
+                            {selectedAuthMethod === 'use-secret' && (
+                                <FormLabelGroup
+                                    label="Secret"
+                                    isRequired={
+                                        values.updatePassword && selectedAuthMethod === 'use-secret'
```

Comment:
Nit: given the conditional rendering the second half of this condition is not necessary.